grid/payments/read-plans/profile.supr

```diff
@@ -0,0 +1,217 @@
+"""
+Get Information About Subscription Plans
+Gets information about subscription plans in a payment system (e.g. PayPal, Stripe...)
+"""
+name = "payments/read-plans"
+version = "1.0.0"
+
+"""
+Get Subscription Plan
+Gets information about a single subscription plan in a payment system.
+"""
+
+usecase GetPlan safe {
+  input {
+    """
+    ID
+    Unique identifier of subscription plan.
+    """
+    planId! string!
+  }
+
+  result {
```

Comment:
suggestion (non-blocking):  The code violates DRY principle.

I propose to define model `Plan` which would be used in `GetPlan` and `ListPlans` use case results. I noticed that `ListPlans` is missing `state` field, I wonder if it is the reason why the `Plan` model is not shared between the use cases?

Reply:
fixed, please one last review check 🙏